src/Umbraco.Web/Models/Mapping/MemberModelMapper.cs

```diff
@@ -96,9 +98,10 @@ public override void ConfigureMappings(IConfiguration config, ApplicationContext
                 .ForMember(dto => dto.HasPublishedVersion, expression => expression.Ignore());
 
             //FROM MembershipUser TO MemberBasic
-            config.CreateMap<MembershipUser, MemberBasic>()
+            config.CreateMap<MembershipUser, MemberBasic>()                
                 //we're giving this entity an ID of 0 - we cannot really map it but it needs an id so the system knows it's not a new entity
                 .ForMember(member => member.Id, expression => expression.MapFrom(user => int.MaxValue))
+                .ForMember(display => display.Udi, expression => expression.Ignore())

```

Comment:
Will a member not have a UDI in case members would ever want to be deployed/transferred or that a some kind of member picker is used?

Not quite sure the difference between a MembershipUser & IMember is here?!